helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -288,6 +280,16 @@ splunk_hec/platform_traces:
     queue_size: {{ .Values.splunkPlatform.sendingQueue.queueSize }}
 {{- end }}
 
+{{/*
+Splunk sending queue
+*/}}
+{{- define "splunk-otel-collector.splunkSendingQueue" -}}
```

Comment:
If persistent queues were added to splunkObservability components, would they be able to use this? If not, we should rename this here and else where.
splunk-otel-collector.splunkSendingQueue -> splunk-otel-collector.splunkPlatformSendingQueue

Reply:
I renamed this one and other occurrences

Reply:
Also I added a changelog entry